src/search-injection/content_script.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import Results from './components/Results'
+import * as constants from './constants'
+import { appendCss, getLocalStorage } from './utils'
+import { SEARCH_CONN_NAME, CMDS } from '../overview/constants'
+
+const handleRender = results => {
+    // The actual function to render the results on screen.
+
+    const renderComponent = async () => {
+        const position = await getLocalStorage(constants.POSITION_KEY, 'above')
+        const containerID = constants.SEARCH_ENGINES.google.container[position]
+        // Gets the container using the passed id
+        const container = document.getElementById(containerID)
+
+        // If re-rendering remove the already present component
+        const component = document.getElementById('memexResults')
+        if (component) component.parentNode.removeChild(component)
+
+        const target = document.createElement('div')
+        target.setAttribute('id', 'memexResults')
+        container.insertBefore(target, container.firstChild)
+
+        // Number of results to limit
+        const limit = constants.LIMIT[position]
+
+        // Render our React component on the target element
+        // Passing this same function so that it can change position
+        ReactDOM.render(
+            <Results
+                results={results.slice(0, limit)}
+                len={results.length}
```

Comment:
`results.length` will always be <=10 because of pagination. Instead, where you call `handleRender`, you can use `payload.searchResult.totalCount` to get the real number (maybe just pass in the entire `payload.searchResult` and pull out the needed args: `const handleRender = ({ docs, totalCount }) => {` )

Reply:
Ah, I was planning to look into this issue, as it never showed me a length greater than 10.
But I don't think the there is a `totalCount` attribute in the payload received.
![image](https://user-images.githubusercontent.com/3889675/37656708-d08af480-2c6e-11e8-99cb-70b0fa3e849d.png)